diff --git a/lexers/LexBash.cxx b/lexers/LexBash.cxx
index 4517df0c..80a74a53 100644
--- a/lexers/LexBash.cxx
+++ b/lexers/LexBash.cxx
@@ -37,6 +37,9 @@
 using namespace Scintilla;
 using namespace Lexilla;
 
+#include "../../../../../../src/syntax/wex/lex-rfw.cpp"
+#include "../../../../../../src/syntax/wex/lex-lilypond.cpp"
+  
 namespace {
 
 #define HERE_DELIM_MAX			256
diff --git a/src/Lexilla.cxx b/src/Lexilla.cxx
index 4a0ce4e5..ddd6ad3b 100644
--- a/src/Lexilla.cxx
+++ b/src/Lexilla.cxx
@@ -131,6 +131,8 @@ extern LexerModule lmR;
 extern LexerModule lmRaku;
 extern LexerModule lmREBOL;
 extern LexerModule lmRegistry;
+extern LexerModule lmLilyPond;
+extern LexerModule lmRFW;
 extern LexerModule lmRuby;
 extern LexerModule lmRust;
 extern LexerModule lmSAS;
@@ -279,6 +281,8 @@ void AddEachLexer() {
 		&lmRaku,
 		&lmREBOL,
 		&lmRegistry,
+		&lmLilyPond,
+		&lmRFW,
 		&lmRuby,
 		&lmRust,
 		&lmSAS,
