diff --git a/lexers/LexBash.cxx b/lexers/LexBash.cxx
index 07dc6c1c..72f37e06 100644
--- a/lexers/LexBash.cxx
+++ b/lexers/LexBash.cxx
@@ -32,6 +32,9 @@
 
 using namespace Scintilla;
 
+#include "../../../../../../src/syntax/wex/lex-rfw.cpp"
+#include "../../../../../../src/syntax/wex/lex-lilypond.cpp"
+
 #define HERE_DELIM_MAX			256
 
 // define this if you want 'invalid octals' to be marked as errors
diff --git a/src/Lexilla.cxx b/src/Lexilla.cxx
index bd15d392..9385ba12 100644
--- a/src/Lexilla.cxx
+++ b/src/Lexilla.cxx
@@ -128,6 +128,8 @@ extern LexerModule lmR;
 extern LexerModule lmRaku;
 extern LexerModule lmREBOL;
 extern LexerModule lmRegistry;
+extern LexerModule lmLilyPond;
+extern LexerModule lmRFW;
 extern LexerModule lmRuby;
 extern LexerModule lmRust;
 extern LexerModule lmSAS;
@@ -273,6 +275,8 @@ void AddEachLexer() {
 		&lmRaku,
 		&lmREBOL,
 		&lmRegistry,
+		&lmLilyPond,
+		&lmRFW,
 		&lmRuby,
 		&lmRust,
 		&lmSAS,
