diff --git a/build/cmake/lib/stc/CMakeLists.txt b/build/cmake/lib/stc/CMakeLists.txt
index f96687d2d8..12ef655f54 100644
--- a/build/cmake/lib/stc/CMakeLists.txt
+++ b/build/cmake/lib/stc/CMakeLists.txt
@@ -165,7 +165,6 @@ target_include_directories(wxscintilla PRIVATE
     ${wxSOURCE_DIR}/src/stc/scintilla/src
     )
 target_compile_definitions(wxscintilla PUBLIC
-    NO_CXX11_REGEX
     __WX__
     SCI_LEXER
     LINK_LEXERS
@@ -189,7 +188,6 @@ wx_lib_include_directories(stc PRIVATE
     ${wxSOURCE_DIR}/src/stc/scintilla/src
     )
 wx_lib_compile_definitions(stc PRIVATE
-    NO_CXX11_REGEX
     __WX__
     SCI_LEXER
     LINK_LEXERS
diff --git a/build/cmake/setup.cmake b/build/cmake/setup.cmake
index 35ba62b64b..8f02be356c 100644
--- a/build/cmake/setup.cmake
+++ b/build/cmake/setup.cmake
@@ -158,7 +158,7 @@ if(wxUSE_STL)
         string functional algorithm vector list
         )
     if(NOT wxTEST_STL)
-        message(FATAL_ERROR "Can't use wxUSE_STL as basic STL functionality is missing")
+#        message(FATAL_ERROR "Can't use wxUSE_STL as basic STL functionality is missing")
     endif()
 
     wx_check_cxx_source_compiles("
index c06a8cc5f0..1c6ddd2e10 100644
--- a/build/cmake/lib/png.cmake
+++ b/build/cmake/lib/png.cmake
@@ -34,7 +34,7 @@ if(wxUSE_LIBPNG STREQUAL "builtin")
             src/png/pngwutil.c
             src/png/arm/arm_init.c
             src/png/arm/filter_neon_intrinsics.c
-            src/png/arm/palette_neon_intrinsics.c
+#            src/png/arm/palette_neon_intrinsics.c
             src/png/intel/intel_init.c
             src/png/intel/filter_sse2_intrinsics.c
             ${PNG_EXTRA_SOURCES}
