diff --git a/build/cmake/lib/stc/CMakeLists.txt b/build/cmake/lib/stc/CMakeLists.txt
index f96687d2d8..12ef655f54 100644
--- a/build/cmake/lib/stc/CMakeLists.txt
+++ b/build/cmake/lib/stc/CMakeLists.txt
@@ -165,7 +165,6 @@ target_include_directories(wxscintilla PRIVATE
     ${wxSOURCE_DIR}/src/stc/scintilla/src
     )
 target_compile_definitions(wxscintilla PUBLIC
-    NO_CXX11_REGEX
     __WX__
     SCI_LEXER
     LINK_LEXERS
@@ -189,7 +188,6 @@ wx_lib_include_directories(stc PRIVATE
     ${wxSOURCE_DIR}/src/stc/scintilla/src
     )
 wx_lib_compile_definitions(stc PRIVATE
-    NO_CXX11_REGEX
     __WX__
     SCI_LEXER
     LINK_LEXERS
diff --git a/build/cmake/setup.cmake b/build/cmake/setup.cmake
index 35ba62b64b..8f02be356c 100644
--- a/build/cmake/setup.cmake
+++ b/build/cmake/setup.cmake
@@ -158,7 +158,7 @@ if(wxUSE_STL)
         string functional algorithm vector list
         )
     if(NOT wxTEST_STL)
-        message(FATAL_ERROR "Can't use wxUSE_STL as basic STL functionality is missing")
+#        message(FATAL_ERROR "Can't use wxUSE_STL as basic STL functionality is missing")
     endif()
 
     wx_check_cxx_source_compiles("
diff --git a/include/wx/msw/winundef.h b/include/wx/msw/winundef.h
index 879a6ef626..feebf2b6eb 100644
--- a/include/wx/msw/winundef.h
+++ b/include/wx/msw/winundef.h
@@ -268,7 +268,7 @@
    #else
    inline int StartDoc(HDC h, CONST DOCINFOA* info)
    {
-      return StartDocA(h, const_cast<DOCINFOA*>(info);
+      return StartDocA(h, const_cast<DOCINFOA*>(info));
    }
    #endif
 #endif
