diff --git a/build/cmake/lib/qa/CMakeLists.txt b/build/cmake/lib/qa/CMakeLists.txt
index 1efa85e255..20ba26ac42 100644
--- a/build/cmake/lib/qa/CMakeLists.txt
+++ b/build/cmake/lib/qa/CMakeLists.txt
@@ -10,4 +10,4 @@
 wx_append_sources(QA_FILES QA)
 
 wx_add_library(wxqa ${QA_FILES})
-wx_lib_link_libraries(wxqa PUBLIC wxxml)
+wx_lib_link_libraries(wxqa PUBLIC)
diff --git a/src/gtk/toplevel.cpp b/src/gtk/toplevel.cpp
index f4fed8eaa5..0e60a0c7d1 100644
--- a/src/gtk/toplevel.cpp
+++ b/src/gtk/toplevel.cpp
@@ -430,7 +430,7 @@ void wxTopLevelWindowGTK::GTKHandleRealized()
                 if ((m_gdkFunc & GDK_FUNC_CLOSE) == 0)
                     layout.Replace("close", empty, false);
 
-                gtk_header_bar_set_decoration_layout(GTK_HEADER_BAR(titlebar), layout);
+                gtk_header_bar_set_decoration_layout(GTK_HEADER_BAR(titlebar), layout.c_str());
             }
 #endif // 3.12
             // Don't set WM decorations when GTK is using Client Side Decorations
